addons/mail/static/src/core/web/full_recipients_list.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="mail.FullRecipientsList">
+        <t t-if="props.thread.recipients.length > 0">
+            <t t-out="getRecipientListToHTML()"/>
+            <button class="o-mail-Chatter-recipientListButton btn btn-link badge rounded-pill border-0 p-1" title="Show all recipients" t-on-click="onClickRecipientList">
+                <i class="fa fa-caret-down"/>
+            </button>
```

Comment:
Note that the BEM convention in discuss we use is `o-moduleName-ComponentName-partName`.
Following this prevents conflicting names and makes traceability easy.
For the root node, if there's one, we usually just put `o-moduleName-ComponentName`.
If there's some classname that acts as modifiers (like changes the style slightly), we simply use `o-modifierName` and combine the BEM classname with `o-modifierName` in the SCSS.

Reply:
I updated the class name following that nomenclature :smiley: I also created a new scss file for the styling of the new component and I moved that rule in there.